### ISSUE TITLE
Replace Progress Pride flag image in Credits page on Feature Homepage Launch branch

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#6286]
### Overview
We need to replace the graphic file presenting the Progress Pride flag on the Credits page, within the feature homepage branch.  The new graphic file is a different file format than the existing file, so the image URL must be updated in the corresponding Credits yml file.  

### Action Items
- [x] Work on this issue will be merged into hackforla/feature-homepage-launch.  For information about working on a feature branch, see https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch
- [x] Download the "New image" from the Resources section below, then rename it `ProgressSQsm.png` and save it in the folder `/assets/images/`
- [x] Delete the existing file `/assets/images/credits/ProgressSQsm.jpg`
- [x] Open the file `_data/internal/credits/diversity-equity-inclusion-secondary.yml` in your IDE and replace
```
image-url: assets/images/credits/ProgressSQsm.jpg
```
with
```
image-url: assets/images/credits/ProgressSQsm.png
```
- [x] Using Docker, view the Credits page (/Credits) and confirm that the appearance of the Credit for the Progress Pride image (text and image) matches the "Current appearance of the Credit for the Progress Pride image" screenshot below, except for the rotation of the image
- [x] Read the Style Guide in the Resource section below and confirm that our usage of the graphic conforms with the guide.  

### Resources/Instructions
<details><summary>New Image</summary>

![Progress Pride Flag by Daniel Quasar](https://github.com/hackforla/website/assets/4952258/23031d6d-f57d-4ea3-847b-6e15fb08c611)
</details> 

<details><summary>Current appearance of the Credit for the Progress Pride image</summary>

<img width="1718" alt="image" src="https://github.com/hackforla/website/assets/68984808/21f3a1f1-1be8-474b-a286-5039147f8efa">
</details> 

Style Guide https://cdn.shopify.com/s/files/1/0589/1676/5845/files/HowToUse-ProgressPrideFlag-DanielQuasar.pdf?v=1629073782

**Comment by ramitaarora** [comment]
i. Availability: Weekdays 9am-2pm
ii. ETA: Wednesday latest

**Comment by ramitaarora** [comment]
I am blocked because I am unable to download the image. I attached a screenshot of the error message.

<img width="1582" alt="Screenshot 2024-02-28 at 9 55 20 AM" src="https://github.com/hackforla/website/assets/104021517/8bbff5e6-1784-4d94-b024-a9769bbdd1ea">

Also, it seems that the original image does not exist in `/assets/images/credits/ProgressSQsm.jpg` and the file `_data/internal/credits/diversity-equity-inclusion-secondary.yml` is gone

**Comment by ExperimentsInHonesty** [comment]
@ramitaarora Please move this issue to the questions/review column and add the label `ready for dev lead`

**Comment by ExperimentsInHonesty** [comment]
@ramitaarora 
1. Not being able to download the image
A: when I open the accordion for New Image above, and right-click on the image, I am able to save as, to save it locally.  I assume you are clicking on the style guide and trying to get it from there.

2. Your comment about the files not existing
>Also, it seems that the original image does not exist in /assets/images/credits/ProgressSQsm.jpg and the file _data/internal/credits/diversity-equity-inclusion-secondary.yml is gone

   A: The files exist in the feature-homepage-launch feature branch.  They do not exist in the gh-pages branch
   see https://github.com/hackforla/website/tree/feature-homepage-launch/assets/images/credits
   see https://github.com/hackforla/website/tree/feature-homepage-launch/_data/internal/credits
   at the top of action items it has a link to instructions for working on a feature branch.

If this answered all of your questions, please remove the ready for dev lead label and put this issue in the in progress column.

**Comment by ramitaarora** [comment]
@ExperimentsInHonesty Thank you so much! It wasn't downloading before but I just tried again and it worked. Maybe it was an issue with my browser. Apologies for my confusion.

Just for future reference, if I am blocked, do I always move it to Questions/In Review right away?

**Comment by ExperimentsInHonesty** [comment]
> Just for future reference, if I am blocked, do I always move it to Questions/In Review right away?

yep! and add the ready for dev lead label
you can also message like you did in the slack channel as well.

**Comment by ramitaarora** [comment]
Progress: Starting issue
Blockers: Unsure of what to update
Availability: Weekdays 9am-2pm
ETA: Eod Monday Mar 11

**Comment by ramitaarora** [comment]
I am blocked - just confused what I should be updating? Where can I find this info? :)

Thanks!

**Comment by jphamtv** [comment]
Hi @ramitaarora, can you provide more details on what you need help with? All the action items are checked above, which usually indicates they are completed, so not sure where you are stuck. 

Did you create your branch from the feature-homepage-launch branch, and not from gh-pages?

**Comment by ramitaarora** [comment]
@jphamtv I actually finished this (on the correct branch) and the pull request was merged, but then got a ping that this issue needed updating. I'm just not sure what needs to be updated haha.

**Comment by jphamtv** [comment]
@ramitaarora Thanks for that info, I see what happened, this issue wasn't closed automatically when your PR #6390 was merged. There's nothing for you to update.

**Comment by ramitaarora** [comment]
@jphamtv Thanks so much for your help!!
